docs/docs/workflows/create.md

```diff
@@ -29,6 +32,51 @@ For details on the flags, consult the command help via `constellation create -h`
 
 *create* stores your cluster's state into a [`terraform.tfstate`](../architecture/orchestration.md#cluster-creation-process) file in your workspace.
 
+</tabItem>
+<tabItem value="terraform" label="Terraform">
+
+:::caution
+
+There is no extensive input validation with Terraform. If unsure, please use the Constellation CLI instead.
```

Comment:
how much effort would it be to add input validation?

Reply:
This mostly depends on how lax we want to be in regards to the instance type aka if we were to allow only a small subset of (C)VM types, then we probably can add validation for this. 
Apart from that, I think the validation is actually quite similar. I also just added some more defaults for AWS.

Reply:
So I guess with the additions you've made, you still feel this warning is needed and helpful for the reader, right?

Reply:
I'm not 100% sure, but since Terraform is the "expert" way, I removed the warning for now. If we see common errors we can add it again.